Libraries/JavaScriptAppEngine/Initialization/InitializeJavaScriptAppEngine.js

```diff
@@ -126,6 +126,10 @@ function setUpAlert() {
   }
 }
 
+function setUpSymbol() {
+  GLOBAL.Symbol = require('es6-symbol');
```

Comment:
@davidaurelio np :) If you'd rather me move the check into RN so it's more clear, I'm happy to do that!